challenges/objects-arrays.js

```diff
@@ -100,22 +136,28 @@ The zoos need a list of all their animal's names (animal_name only) converted to
 */
 
 const lowCaseAnimalNames = [];
```

Comment:
Remember that the .map() method creates a new array for you, so you don't need to push items into an empty array.